### ISSUE TITLE
Strange prompt output at the start of the gif

### ISSUE THREAD

**Issue by knpwrs** [charmbracelet/vhs#39]
I attempted to put together a gif to demo my crate, [woodo](https://lib.rs/woodo): https://github.com/knpwrs/woodo

Here is my script:

```tape
Output woodo.gif

Set FontSize 32
Set Width 800
Set Height 1200

Type woodo
Sleep 500ms
Enter
Sleep 500ms
Type sudo woodo
Sleep 500ms
Enter
Type lolnope
Enter
Sleep 5s
```

Here is the output gif:

![woodo](https://user-images.githubusercontent.com/174864/198338724-4651c97c-b987-409f-b74f-2de28dc86e74.gif)

The gif starts out with 11 frames, ending with:

![image](https://user-images.githubusercontent.com/174864/198338936-ff1b8e21-29cd-404c-8f0c-b8676f0db992.png)

Before transitioning to:

![image](https://user-images.githubusercontent.com/174864/198338978-119be033-7697-43c3-9388-f77fa66e5112.png)

**Comment by maaslalani** [comment]
@knpwrs, can you add the following to your `.bashrc`?

```
export BASH_SILENCE_DEPRECATION_WARNING=1
```

If this works for you, I'll add it by default in VHS so that people don't have to add it manually.

**Comment by joglr** [comment]
I am experiencing something similar on Windows. @knpwrs How are you looking at the individual frames?
![demo](https://user-images.githubusercontent.com/1959615/198898070-45c4d035-41d0-4923-aff2-d64686e154f4.gif)

**Comment by maaslalani** [comment]
As a temporary measure, until we get this fixed, I believe this adding a `Hide` + `Sleep` should skip the frames for you at the beginning.

```
Hide Sleep 1s Show

# Start ...
```

**Comment by caarlos0** [comment]
I think I might have improved that in #55

**Comment by knpwrs** [comment]
> How are you looking at the individual frames?

When you open `.gif` files in `Preview.app` on macOS all of the individual frames show up in the left-hand thumbnail view.

**Comment by caarlos0** [comment]
You can also do `Set Output frames/` (trailing slash is important I think) and vhs will output all frames as png in a `frames` folder.

**Comment by enrichman** [comment]
I got the same issue on a Mac, this is the first frame of the example tape:

<img width="882" alt="image" src="https://user-images.githubusercontent.com/1763949/199923279-b867b5c6-c741-4acb-8e2a-0df020f46c07.png">

**Comment by bbkane** [comment]
I'm getting extra frames too, with `zsh` on Macos:

<img width="1219" alt="image" src="https://user-images.githubusercontent.com/6081085/205473432-6a7b5594-256a-40ba-8cd5-bd324e1af975.png">

**Comment by transitive-bullshit** [comment]
Same problem here; no matter what I do or which shell I use, these `vhs`-specific shell commands appear at the start of the video.

https://user-images.githubusercontent.com/552829/205569182-0183a7a7-63e8-4c43-8330-a0409749c314.mp4

My current attempt:
```
Output media/demo.mp4

Hide
Require npx

#Set Shell bash
Set FontSize 17
Set Width 1200
Set Height 600
Set Padding 24
#Set LoopOffset 75%

Show

Sleep 1s

Type "npx tsx src/example.ts"

Sleep 500ms
Enter
Show

Sleep 5s
Sleep 5s
```

**Comment by bashbunni** [comment]
I know this is from a while back, but I just tried it on my Mac after installing vhs with `brew` and it *seems* to be fixed. Is anyone still having this issue?

**Comment by bbkane** [comment]
After commenting out my workaround ( lines 16-19) I just retried it with [demo.tape](https://github.com/bbkane/fling/blob/f9a2bcec8af054c03cdda6d60e6233fc11b92645/demo.tape) and still saw the extraneous text I mentioned in above comments.

```
$ vhs --version
vhs version v0.2.0 (6f7cc81)
```

**Comment by caarlos0** [comment]
@bbkane that might be it, yes, if so, this should help: https://github.com/charmbracelet/vhs/pull/212

**Comment by bbkane** [comment]
I can retry after that's merged, thanks @caarlos0 ! If I get time, I'll also try with an unconfigured zsh and see what happens

**Comment by caarlos0** [comment]
sounds good, merged @bbkane

**Comment by bashbunni** [comment]
I'm actually seeing this exact message in a commit for vhs `Setup()` func 
https://github.com/charmbracelet/vhs/commit/f21a959d2b5c6687e290c317e8f22cbbbced12b5

**Comment by caarlos0** [comment]
@bashbunni yeah that cmd runs before right after starting zsh... my guess is that the user/system config is taking a bit too long to load, and then the `clear` takes a little too long to exec, and we see those frames.

If that's the issue, my PR should fix or at least help with that... it prevents loading user & system configs, and also uses another way to set prompt.

**Comment by graynk** [comment]
I've just installed it from brew and the issue is still present. Though the workaround with `Hide Sleep 5s Show` works

**Comment by caarlos0** [comment]
the fixes I made are not released yet I think...
